src/csharp/Intel/Iced/Intel/Assembler/AssemblerMemoryOperand.cs

```diff
@@ -84,7 +86,7 @@ namespace Iced.Intel {
 		/// <summary>
 		/// Gets the mask associated with this operand.
 		/// </summary>
-		public readonly AssemblerOperandFlags Flags;
+		internal readonly AssemblerOperandFlags Flags;
```

Comment:
Should `IsBroadcast` and `ToMemoryOperand()` below also be made internal?

Reply:
I can make them internal.

Reply:
Hm, for `ToMemoryOperand` it is actually quite practical to have it public. Otherwise there is no way to compute a MemoryOperand easily.

Reply:
OK, keep it public.

Reply:
Same for `IsBroadcast`, I believe it's preferable to keep it, otherwise it can forbid simple scenarios where you want to automate changes. You would have to use `ToMemoryOperand` to access all the information, and then recreate by using operations.